packages/tools/src/ui/tools-ui-overview-item.tsx

```diff
@@ -9,9 +9,12 @@ export function ToolsUiOverviewItem({ tool }: { tool: Tool }) {
   return (
     <Item
       asChild
-      className={cn({
-        'text-muted-foreground': tool.comingSoon,
-      })}
+      className={
+        'rounded-xl ' +
+        cn({
+          'text-muted-foreground': tool.comingSoon,
+        })
+      }
```

Comment:
By using the `cn` function you don't have to concatenate strings anymore. It's built for this exact use case

```ts
cn('rounded-xl', {
  'text-muted-foreground': tool.comingSoon,
})
```